examples/egl_common.h

```diff
@@ -0,0 +1,26 @@
+#ifndef _EGL_COMMON_H
+#define _EGL_COMMON_H
+#endif
+
+#include <stdbool.h>
+#include <wayland-client.h>
+
+#include <EGL/egl.h>
+#include <EGL/eglext.h>
+
+struct egl_ctx {
+	EGLDisplay display;
+	EGLConfig config;
+	EGLContext context;
+
+	PFNEGLGETPLATFORMDISPLAYEXTPROC eglGetPlatformDisplayEXT;
+	PFNEGLCREATEPLATFORMWINDOWSURFACEEXTPROC eglCreatePlatformWindowSurfaceEXT;
```

Comment:
Since this is only intended to be used by examples, can we use global variables for these instead?

Reply:
Done! Just FYI, I've taken `eglGetPlatformDisplayEXT` out of `egl_common.h` as it's only used by `egl_common.c` to create the `EGLDisplay` for the examples that rely on it.